src/main/webapp/enableJS.jsp

```diff
@@ -1,33 +1,4 @@
 <noscript>
-<style>
```

Comment:
You should leave out changes from the other pull request, which is unrelated to this issue.